### ISSUE TITLE
Include translation project in installer

### ISSUE THREAD

**Issue by andersmelander** [andersmelander/better-translation-manager#8]
In order to enable user translation of BTM, the required files should be made available for install as an optional component in the installer. The files include:

* amTranslationManager.xlat
* amTranslationManager.drc

The drc file is not needed for translation but is required if the user is to generate resource modules. 

---

Depends on issue #7
